lib/decorators/metadata.js

```diff
@@ -1,28 +1,35 @@
 const Cacheable = require('./cacheable');
+const { capitalize } = require('lodash');
 
 module.exports = settings => {
 
   const { Establishment, Profile } = settings.models;
   const cache = Cacheable();
 
   return c => {
-
     const promises = [
       c.data.establishmentId && cache.query(Establishment, c.data.establishmentId),
       c.data.subject && cache.query(Profile, c.data.subject),
       c.data.changedBy && cache.query(Profile, c.data.changedBy)
     ];
 
+    if (c.data.id && c.data.model) {
+      const modelName = c.data.model === 'pil' ? c.data.model.toUpperCase() : capitalize(c.data.model);
```

Comment:
not sure if this scales to _all_ the models - thinking `ProjectVersion` and `AsruEstablishment`